include/glow/Graph/Graph.h

```diff
@@ -224,17 +225,16 @@ class Function final : public Named {
   /// @name High-level, operation-level IRBuilder.
   ///@{
 
-  ConvolutionNode *createConv(llvm::StringRef name, NodeValue input,
-                              NodeValue filter, NodeValue bias, TypeRef outTy,
-                              llvm::ArrayRef<unsigned_t> kernels,
-                              llvm::ArrayRef<unsigned_t> strides,
-                              llvm::ArrayRef<unsigned_t> pads,
-                              unsigned_t group);
+  llvm::Expected<ConvolutionNode *>
+  createConv(llvm::StringRef name, NodeValue input, NodeValue filter,
```

Comment:
All others will changed to llvm::Expected as well in follow up diffs. I'm going to do them in smallish batches so that it's more reviewable. I just included this one to show how it will go.

Reply:
as we discussed offline, can you put this description in the PR and outline future work